.environment/gitleaks/gitleaks-config.toml

```diff
@@ -16,12 +16,16 @@ title = "PRIME ReportStream Gitleaks Configuration"
         '.terraform/providers/',
         'frontend-react/build/',
         'frontend-react/node_modules/',
+        'frontend/node_modules/',
+        'frontend/dist',
+        'frontend/src/assets',
```

Comment:
This I'm not totally comfortable with; can this instead be done by file type? What if we put JS in there or anything else...? We'd want the JS to be scanned, but -for instance- not fonts or JPEGs.

Reply:
Frontend is going away by end of this sprint -- lets not make this too complex

Reply:
Discussed in person, @mry-usds is correct